lib/l10n/intl_fr.arb

```diff
@@ -18,109 +18,109 @@
     "placeholders_order": [],
     "placeholders": {}
   },
-  "email": "email",
+  "email": "courriel",
   "@email": {
     "type": "text",
     "placeholders_order": [],
     "placeholders": {}
   },
-  "password": "password",
+  "password": "mot de passe",
   "@password": {
     "type": "text",
     "placeholders_order": [],
     "placeholders": {}
   },
-  "login": "Login",
+  "login": "Se connecter",
   "@login": {
     "type": "text",
     "placeholders_order": [],
     "placeholders": {}
   },
-  "login_text_login_to_continue": "Please login to continue",
+  "login_text_login_to_continue": "Veuillez vous connecter pour continuer",
   "@login_text_login_to_continue": {
     "type": "text",
     "placeholders_order": [],
     "placeholders": {}
   },
-  "unknown_error_login_message": "Unknown error occurred, please try again",
+  "unknown_error_login_message": "Une erreur inconnue s'est produite. Veuillez réessayer.",
   "@unknown_error_login_message": {
     "type": "text",
     "placeholders_order": [],
     "placeholders": {}
   },
-  "search_folder": "Search folder",
+  "search_folder": "Dossier de recherche",
```

Comment:
```suggestion
  "search_folder": "Rechercher dans vos dossiers",
```

Reply:
This is testing. I will not merge this PR. BTW, please contribute directly on Weblate :).